packages/lists/init.lua

```diff
@@ -44,10 +44,10 @@ local styles = {
    enumerate = {
       { display = "arabic", after = "." },
       { display = "roman", after = "." },
-      { display = "alpha", after = ")" },
+      { display = "alpha", after = "." },
```

Comment:
I don't get what these various changes on the lists have to do with the issue "normalizing command vs. whitespace handling"?

Reply:
They don't, they should be broken out into another commit, but I was going to ask you about them. The sequence didn't make any sense to me. I already removed the arbitrary limit of 6 levels and set it to just repeat from the set, but the 2-1-2-1 alternation of trailing characters didn't make sense to me. Either `...)))` or `.).).)` would make sense, but where did `..))).` come from? I looked around at some word processors and such and couldn't find a precedent for that. Was this something that came out of TeX?

I'm not set on changing it, but as long as all the alignment is getting overhauled I thought it would be a better time than most to make it more logical.

Reply:
I checked quickly and actually I don't really care as far as concerned: whether using Markdown or Djot, these define the list separators (dots, parens, etc.) based on their own input syntax, so don't use the default styling. I don't really care for my "resilient" lists, since styles can be overidden at the users convenience. In other terms, I don't have any strong opinion here on what SILE's enumeration list should use by default.